### DISCUSSION TITLE
Osiris vs OsirisAndExtra

### DISCUSSION THREAD

**Discussion by testingthisout12345** [notgoodusename/OsirisAndExtra#577] [General]
Was wondering if there's any major difference between these two "forks" other than the added features? Both seem to get regular "updates" still. To me it looks like OsirisAndExtra is based on older code from Osiris. 

If I were to fork one for my personal use should I use the original Osiris source & add features to it, or should I use the OsirisAndExtra one and remove/add features to it?

To me the Osiris source looks more organised. Dunno.

**Comment by notgoodusename** [comment]
That depends on what you want the cheat to have.
The difference is that OsirisAndExtra has ragebot, doubletap, antiaim, fakelag, fake angles, a better legit aimbot, backtrack triggerbot, more misc features (playerlist, svpure bypass, custom models, jump stats, edge bug, etc)
And i can't speak about how cluttered it is since for me it isn't, but that's because I'm the creator

**Reply by testingthisout12345** [reply]
Thanks for your answer. I think I'll go with this one instead of the base one.

And uhm... It's not cluttered. It's just so different from the other one. I spent a good week on the other base & got used to it so when I opened this one up, I was a bit lost at first. All good now. Sorry if it came out wrong, haven't slept at all.

**Reply by notgoodusename** [reply]
Yeah, I get what you mean
